packages/suite/src/components/suite/NavigationBar/components/DeviceSelector/index.tsx

```diff
@@ -128,10 +130,18 @@ const DeviceSelector = (props: React.HTMLAttributes<HTMLDivElement>) => {
         }
     }, [countChanged]);
 
+    useEffect(() => {
+        // if the device status changes, show wallet state (dis/connected) as text for 2 seconds
+        setShowTextStatus(true);
+        setTimeout(() => {
+            setShowTextStatus(false);
+        }, 2000);
```

Comment:
you are missing a cleanup function for clearing a timeout, it probably is not a big deal in this case, but it is a good practise anyway.
```
return () => {
    // code for clearing timeout
}
```

some reading material https://reactjs.org/docs/hooks-effect.html#effects-with-cleanup

I am also wondering if it could happen that setShowTextStatus will be triggered once the component is already unmounted, it doesn't seem likely as this component shouldn't be getting unmounted anyway (if it happens you will see some warning in the console about updating state of unmounted component).